_global/js/card.js

```diff
@@ -1,5 +1,14 @@
 var quickCards = document.querySelectorAll('quick-card');
 
+window.onload = function startMain(){
+    const cards = document.querySelectorAll('card');
+    
+	for (i = 0; i < cards.length; ++i) {
+		var html = cards[i].innerHTML;
+		cards[i].innerHTML = replaceIcon(html);
+	}
```

Comment:
This can be simplified to:

```suggestion
	for (var card of cards) {
		card.innerHTML = replaceIcon(card.innerHTML);
	}
```